olm/README.md

```diff
@@ -0,0 +1,27 @@
+# Creating new OLM artifacts
+
+Script `update-nightly-olm-files.sh` is used for that
+
+Pre-requisites
+ - [yq/https://pypi.org/project/yq](https://pypi.org/project/yq) and not [http://mikefarah.github.io/yq/](http://mikefarah.github.io/yq/)
```

Comment:
link label is "yq/https://pypi.org/project/yq" ... perhaps you just want "yq" ?

Reply:
Or `https://pypi.org/project/yq` simply ?